### PR TITLE
Merge-Request-Worker-Phase-1

### PR DESCRIPTION
**Pull request by abhinavbajpai2012**
Adding gitlab-merge-request-worker with the following functionalities -

Query and store merge requests
Query and store MR labels
Query contributors (using worker_base)
Query MR events
Query MR comments/Notes

### CODE REVIEW

**Comment by sgoggins** [comment]
@ccarterlandis / @gabe-heim : I see a number of files in here that seem like they are from the old GSOC branch. Jupyter Notebooks, a new issues.py routes file, etc. 

Tell me if something is looking wrong here.

**Comment by sgoggins** [comment]
Yeah, there are commits all the way back to March. @abhinavbajpai2012 : I wonder if its possible for you to make a new fork, or if we should make you a new branch from `dev` and have you add files to that... ?

**Comment by mrsaicharan1** [comment]
@sgoggins I guess @abhinavbajpai2012 was done with the merge request worker.. might this be an extension to the previous one?

File: frontend/src/router.ts [hunk 1]
```diff
@@ -31,8 +31,41 @@ import InspectInsight from './views/InspectInsight.vue';
 import RiskMetrics from "@/views/RiskMetrics.vue";
 import NProgress from "nprogress";
 import SlackConfig from './views/SlackConfig.vue';
+import Login from './views/login.vue';
```
Comment (sgoggins) [inline frontend/src/router.ts:34]: WE don't want to expose login.vue in the main branch.

File: frontend/src/router.ts [hunk 2]
```diff
@@ -31,8 +31,41 @@ import InspectInsight from './views/InspectInsight.vue';
...
+import Secure from './views/secure.vue';
```
Comment (sgoggins) [inline frontend/src/router.ts:35]: probably not secure.vue either, but I am less sure what this is.

File: frontend/src/views/secure.vue [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<template>
```
Comment (sgoggins) [inline frontend/src/views/secure.vue:1]: OK ... let me know what these vues are and how they work @mrsaicharan1

File: workers/Adarsh-NP/database_demo/commitdata_collection.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import os, json, requests
```
Comment (sgoggins) [inline workers/Adarsh-NP/database_demo/commitdata_collection.py:1]: I think we don't want this file. The Adarsh-NP folder should be deleted from your branch.

File: workers/Adarsh-NP/database_demo/commits_database.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
+'''Python flask application for creating and inserting data into the commits table'''
```
Comment (sgoggins) [inline workers/Adarsh-NP/database_demo/commits_database.py:1]: Delete folder from your branch. Adarsh-NP

File: workers/IIZODIACII/__init__.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
+from .models import db, Contributor
```
Comment (sgoggins) [inline workers/IIZODIACII/__init__.py:1]: Delete folder from your branch: IIZODIACII

File: frontend/src/router.ts (hunk 2 above)
Comment (sgoggins) [reply frontend/src/router.ts:35]: @abhinavbajpai2012 : Can you get rid of this line in your branch? 
login.vue and 
secure.vue 

are not files we want in the main folder frontend yet.

File: frontend/src/views/secure.vue (hunk 3 above)
Comment (sgoggins) [reply frontend/src/views/secure.vue:1]: @abhinavbajpai2012 : Tagged teh wrong person #Doh

File: notebooks/Isha-AnomalyDetection.ipynb [hunk 7]
```diff
@@ -0,0 +1,1452 @@
+{
```
Comment (sgoggins) [inline notebooks/Isha-AnomalyDetection.ipynb:1]: do a PR FROM `gsoc-dev`, with arrow pointed at your branch to clean these notebooks up, @abhinavbajpai2012

File: workers/Adarsh-NP/api_demo/app.py [hunk 8]
```diff
@@ -0,0 +1,19 @@
+'''Simple python script to interact with the apis'''
```
Comment (sgoggins) [inline workers/Adarsh-NP/api_demo/app.py:1]: @abhinavbajpai2012 : Is this something you mean to deploy now, or is it from the earlier period and should be cleaned up? If so, the PR FROM `gsoc-dev` arrow pointed at your branch (opposite of this PR) FIRST, *should* clean that up.

**Comment by sgoggins** [comment]
@ccarterlandis : Do you know if the PRs are failing TRAVIS here because there's something not in tox.ini that needs to be there? There are new routes in this PR. 

```
ERROR: InvocationError for command /home/travis/build/chaoss/augur/.tox/py36-metric-routes/bin/python tests/test_routes/runner.py (exited with code 1)
```

**Comment by abhinavbajpai2012** [comment]
Will be cleaning this. I have reforked and made the changes required.

**Pull request closed without merging**